tests/unit/transformers/test_datetime.py

```diff
@@ -430,6 +430,39 @@ def test__reverse_transform_datetime_format_with_strftime_formats(self):
         if 'windows' not in platform.system().lower():
             pd.testing.assert_series_equal(output, expected)
 
+    def test__reverse_transform_datetime_format_with_nans(self):
+        """Test the ``_reverse_transform`` method returns the correct datetime format with nans."""
+        # Setup
+        ute = UnixTimestampEncoder()
+        ute.datetime_format = '%b %-d, %Y'
+        transformed = np.array([1.5778368e+18, 1.5805152e+18, np.nan])
+        ute._dtype = 'object'
+        ute.null_transformer = NullTransformer('mean')
+
+        # Run
+        output = ute._reverse_transform(transformed)
+
+        # Assert
+        expected = pd.Series(['Jan 1, 2020', 'Feb 1, 2020', np.nan])
+        if 'windows' not in platform.system().lower():
+            pd.testing.assert_series_equal(output, expected)
```

Comment:
I see we do this elsewhere but do you know why we skip for windows?

Reply:
Not sure. I can't test locally since I don't have a windows machine, but I can try removing it and seeing what happens to the job